scripts/download_file.py

```diff
@@ -0,0 +1,156 @@
+
+import argparse
+import urllib.request
+import urllib.error
+import os
+import hashlib
+import sys
+
+
```

Comment:
`def parse_args(): argparse.Namespace` so that this does not run every time the file executes